SECURITY.md

```diff
@@ -10,11 +10,14 @@ security.
 ### Marker traits
 
 Rand provides the marker traits `CryptoRng`, `TryCryptoRng` and
-`CryptoBlockRng`. Generators implementing one of these traits and used in a way
-which meets the following additional constraints:
-
--   Instances of seedable RNGs (those implementing `SeedableRng`) are
-    constructed with cryptographically secure seed values
+`CryptoBlockRng`. Generators (RNGs) implementing one of these traits which are
+used according to these additional constraints:
+
+-   If the RNG implements `Default`, it may be default-constructed
+-   If the RNG implements `SeedableRng`, it may be constructed and seeded using
+    `SeedableRng::from_seed` with a cryptographically secure seed value
+-   If the RNG implements `SeedableRng`, it may be constructed and seeded from
+    another RNG which is itself cryptographically secure
```

Comment:
Am I reading this correctly ...
Default means it will be seeded securely and automatically without any work by the caller.
SeedableRng relies on the caller providing a seed that is cryptographically secure from some other source, possibly from another RNG.

If so maybe this is more clear to have one SeedableRng bullet...
```suggestion
-   If the RNG implements `Default`, it may be default-constructed
-   If the RNG implements `SeedableRng`, it may be constructed and seeded using
    `SeedableRng::from_seed` with a cryptographically secure seed value, possibly
     from another RNG which is itself cryptographically secure
```

Reply:
`OsRng` and `ThreadRng` can be default constructed and be secure.

Another way to read this is that a `CryptoRng` should not support `Default` if the result would be insecure.

I admit that there is potential for confusion here. Perhaps I should add to the first point:

> Note that generators should only implement `Default` where a default-constructed instance is no more predictable than a securely seeded instance; for example `OsRng` (which is a stateless) supports `Default` construction.